### PR TITLE
Minor fixes to strings

### PR DESCRIPTION
**Pull request by triclambert**

### CODE REVIEW

**Comment by triclambert** [comment]
retest this please

File: universe/config.json [hunk 1]
```diff
@@ -69,12 +69,12 @@
 				"data_center_url": {
 					"id": "http://cassandra/docs/mesosphere.com/service/data_center_url",
 					"type": "string",
-					"description": "The URL of from which the the data center for the cluster managed by this service instance will server its data center information."
+					"description": "The URL from which the data center for the cluster managed by this service instance will server its data center information."
```
Comment (sascala) [inline universe/config.json]: Should it be "will _serve_ its data center information"?

Comment (triclambert) [inline universe/config.json]: Another good catch.  I have opened a separate ticket to comb through all of these messages...  I'm sure there are more.

**Comment by triclambert** [comment]
retest this please

**Comment by sascala** [comment]
retest this please

**Pull request merged**